SparkPost/2web2ui
# FE-1110 - Incorporate UX feedback for the A/B Testing feature

[FE-1110](https://jira.int.messagesystems.com/browse/FE-1110)

### What Changed
- Details button was already fixed in a previous PR!
- Removed use of icons in A/B testing popover
- The "View Template" button now has a right chevron
- The "Remove Variant" button now has a remove icon (a11y requirement missed in the A/C)
- Fixed color of info icon to use blue color

### How To Test
- Sign in to the usual test account
- Verify the above on the following route `/ab-testing/ab_cn_dev_test/2`
- Verify the above on the following route `http://localhost:3100/ab-testing/cutover_test_e/1`

### To Do
